src/ServiceControlInstaller.Engine/Accounts/UserAccount.cs

```diff
@@ -27,7 +27,12 @@ public string QualifiedName
             }
         }
 
-        
+        // ReSharper disable once InconsistentNaming
+        static string LocalizedNTAuthority()
+        {
+            var localSystem = new SecurityIdentifier(WellKnownSidType.LocalSystemSid, null);
+            return localSystem.Translate(typeof(NTAccount)).Value.Split('\\')[0];
```

Comment:
Are we OK to assume that the current user has the ControlPrincipal security permission? https://msdn.microsoft.com/en-us/library/system.security.principal.securityidentifier.translate(v=vs.110).aspx#Anchor_2